lib/business_time/parsed_time.rb

```diff
@@ -19,10 +19,22 @@ def self.parse(time_or_string)
       new(time.hour, time.min, time.sec)
     end
 
+    def strftime(option)
+      to_time.strftime(option)
+    end
+
     def to_s
       "#{hour}:#{min}:#{sec}"
     end
 
+    def to_time
+      Time.parse(to_s)
+    end
+
+    def to_datetime
+      DateTime.parse(to_s)
+    end
```

Comment:
[CONCERN] I'm a little concerned by this. Like `DateTime`, `Time` also holds a Date and a Time. The default parse behavior is to assume today's current date and the current time zone, but there in lies the concern. If the time zone is not what you're expecting, would you always get the expected output?